projects/core/src/model/configurator.model.ts

```diff
@@ -120,6 +120,7 @@ export namespace Configurator {
     STRING = 'string',
     AUTO_COMPLETE_CUSTOM = 'input_autocomplete',
     IMAGE_MULTI_SELECT = 'image_multi_select',
+    SINGLE_SELECTION_IMAGE = 'single_selection_image',
```

Comment:
We should have the same naming style for single and multi select image.
So we have to adapt either IMAGE_MULTI_SELECT => MULIT_SELECTION_IMAGE or SINGLE_SELECTION_IMAGE => IMAGE_SINGLE_SELECT.
Which one we use is not important, but they should be consistent

Reply:
We will rename the multi select component, as this PR is about single select, we merge and change the multi select in seperate branch